src/components/pages/ProfileSelect/RenderProfileSelect.js

```diff
@@ -120,7 +119,7 @@ const RenderProfileSelect = props => {
                   },
                   () => ({
                     validator(rule, value) {
-                      const x = bc.compareSync(value, selected.PIN);
+                      const x = (value === selected.PIN);
```

Comment:
`selected.PIN` is a hashed value, so I think we may need to revert this portion and keep bcrypt.  Currently, with this change, typing in the correct PIN will return falsey, so I believe `const x = bc.compareSync(value, selected.PIN);` is still necessary here

Reply:
plant the new seeds to fix your false problem